source/browser-support/index.html.md.erb

```diff
@@ -0,0 +1,66 @@
+---
+title: Browser support
+weight: 100
+---
+
+# Browser support
+
+To make sure the public can successfully access and use government services, regardless of the browser they're using, the Design System team has divided browsers into 4 grades. Each grade shows the level of support we will provide.
+
+You can see more information about [how we provide support for different browsers](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/browser-support.md) in our GitHub documentation.
```

Comment:
**note** The link needs a PR adding the full version of the docs to the govuk-frontend site to have been merged.